public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -61,14 +61,19 @@ export const LokiQueryBuilderOptions = React.memo<Props>(
       }
     }
 
-    let queryType = query.queryType ?? (query.instant ? LokiQueryType.Instant : LokiQueryType.Range);
-    let showMaxLines = isLogsQuery(query.expr);
+    function onStepChange(e: React.SyntheticEvent<HTMLInputElement>) {
+      onChange({ ...query, step: e.currentTarget.value.trimEnd() });
```

Comment:
You could just:

```suggestion
      onChange({ ...query, step: trim(e.currentTarget.value) });
```
And remove any other space.